### ISSUE TITLE
add support for multiple projects

### ISSUE THREAD

**Issue by dsikkema** [platformsh/platformsh-cli#490]
The idea is, if I have multiple remote projects represented in the same directory as multiple remotes of my git repository, there should be support for interacting with more than one of them. Right now something like the command "environment:ssh" only works against the "main" project, and there's no way to use that specific command to ssh into other projects.

**Comment by pjcdawkins** [comment]
You can do
```
platform ssh -p projectid
```
but maybe I'm missing something

**Comment by dsikkema** [comment]
you're right I updated the issue to clarify. Also FYI this is an issue I discussed with @Kazanir so he has some context.
